src/Pages/Register/RegisterMenu/RegisterMenu.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import styled from 'styled-components';
+import {
+  RegisterMenuBox,
+  RegistImageBox,
+  ImageBoxList,
+  ImageNumber,
+  ImageTitle,
+  FormInput,
+  MenuDetailBox,
+  MenuDetailList,
+  MenuDetailSpan,
+  FormButton,
+} from '../StyleData/StyleData';
+
+const RegisterMenu = ({
+  storeInput,
+  handleStoreInput,
+  handleMenuList,
+  menuImageList,
+}) => {
+  const imageListCheck = menuImageList.filter(Boolean).length;
+  return (
+    <>
+      <RegisterMenuBox>
+        <ImageBoxList className="menu plr20">
+          <ImageTitle>메뉴 이미지</ImageTitle>
+          <FormInput
+            name="menuImage"
+            className="text w100"
+            type="text"
+            value={storeInput.menuImage}
+            onChange={handleStoreInput}
+          />
+        </ImageBoxList>
+        <MenuDetailBox>
+          {inputList.map((input, index) => (
+            <MenuDetailList key={index}>
+              <MenuDetailSpan className="mr5">{input.title}:</MenuDetailSpan>
+              <FormInput
+                name={input.name}
+                className="text w50"
+                type="text"
+                placeholder="내용을 작성해주세요"
+                value={storeInput[input.name]}
+                onChange={handleStoreInput}
+              />
+              <MenuDetailSpan className="ml5">({input.sub})</MenuDetailSpan>
+            </MenuDetailList>
+          ))}
+        </MenuDetailBox>
+        <FormButton
+          className="menuBtn"
+          onClick={e => {
+            handleMenuList(e);
+          }}
```

Comment:
```suggestion
          onClick={handleMenuList}
```
불필요한 함수선언인 것 같습니다 :)

Reply:
앗.. 수정하겠습니다!